packages/cli/src/commands/destroy/page/__tests__/page.test.js

```diff
@@ -28,18 +28,40 @@ beforeEach(() => {
   })
 })
 
-test('destroys a page and route', async () => {
+test('destroys page files', async () => {
   const unlinkSpy = jest.spyOn(fs, 'unlinkSync')
   const t = tasks({ name: 'About' })
   t.setRenderer('silent')
 
-  return t.run().then(() => {
-    // Make sure all generated files were destroyed.
+  return t._tasks[0].run().then(() => {
```

Comment:
@cannikin actually pretty straightforward! Didn't realize individual sub-tasks are also wrapped into `Task` class and runnable.

Destroyers never fail, they just skip the work that cannot be done. We can safely call them multiple times in a row for the same component.